backend/.env.template

```diff
@@ -0,0 +1,7 @@
+DB_USER="postgres"
+DB_PASS="password"
```

Comment:
should this pw be stored in plaintext?

Reply:
this is the .env template, deva are expected to create a .env and populate it with the actual info